src/test/groovy/com/cloudbees/plugins/flow/ConcurrencyTest.groovy

```diff
@@ -97,20 +102,21 @@ class ConcurrencyTest extends DSLTestCase {
 
         def concjob2 = createBlockingJob("concjob2", f1)
 
-        BuildFlow flow = new BuildFlow(Jenkins.instance, getName())
+        BuildFlow flow = new BuildFlow(Jenkins.instance, name.getMethodName())
         flow.concurrentBuild = true;
         flow.dsl = """  build("concjob2")  """
+        flow.onCreatedFromScratch() // need this to updateTransientActions 
 
-        def sfr1 = flow.scheduleBuild2(0)
+        def sfr1 = flow.scheduleBuild2(0, new Cause.UserIdCause())

```

Comment:
hmmmm, well, the first argument is supposed to be the `quietPeriod`, I'm not sure it should be `0` based on the test description.